lib/B/Deparse.t

```diff
@@ -1879,10 +1879,10 @@ CORE::do({});
 >>>>
 () = (return 1);
 () = (return ($1 + $2) * $3);
-() = (return ($a xor $b));
+() = (return $a ^^ $b);
```

Comment:
Are you sure that's correct? Isn't `return` tighter than `^^`?

Reply:
`return` is a list operator with regular (low) precedence. Only super-low precedence operators can escape it (i.e. `and`, `or`, `xor`):
```
$ perl -MO=Deparse -e 'return ($a || $b)'
return $a || $b;
-e syntax OK
```
(`^^` has the same precedence as `||`.)